experiment/build/gcb_build.py

```diff
@@ -160,7 +111,12 @@ def get_build_config_file(filename: str) -> str:
 
 def build_fuzzer_benchmark(fuzzer: str, benchmark: str) -> bool:
     """Builds |benchmark| for |fuzzer|."""
-    if benchmark_utils.is_oss_fuzz(benchmark):
-        _build_oss_fuzz_project_fuzzer(benchmark, fuzzer)
-    else:
-        _build_benchmark_fuzzer(benchmark, fuzzer)
+    substitutions = {
+        '_BENCHMARK': benchmark,
+        '_FUZZER': fuzzer,
+    }
+    config_file = get_build_config_file('fuzzer.yaml')
+    config_name = '{benchmark}-fuzzer-{fuzzer}'.format(benchmark=benchmark,
```

Comment:
Shouldn't this be benchmark-{benchmark}-fuzzer-{fuzzer}

Reply:
Done.